prepare.sh

```diff
@@ -0,0 +1,11 @@
+# move mathjax to static folder for our hacky loading.
```

Comment:
Rename file, as described above. And perhaps add in the code that updates the perseus submodule to the latest.